tests/integration/components/category-list-test.js

```diff
@@ -0,0 +1,24 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('category-list', 'Integration | Component | category list', {
+  integration: true
+});
+
+test('it renders', function(assert) {
+  // Set any properties with this.set('myProperty', 'value');
+  // Handle any actions with this.on('myAction', function(val) { ... });
+
+  this.render(hbs`{{category-list}}`);
+
+  assert.equal(this.$().text().trim(), '');
```

Comment:
Oh wait, we actually have some tests :D Using Travis might not a bad idea after all.